.github/workflows/firebase_deploy.yaml

```diff
@@ -0,0 +1,22 @@
+name: Continuous deployment to firebase hosting
+
+on:
+  push: 
+    branches: [ cmd-f ]
```

Comment:
Is this a wildcard?

Reply:
Are you talking about the array for `branches`? That's just the list of branches the workflow is subscribed to, currently it only has `cmd-f` tho